docs/Installation/Installation.md

```diff
@@ -371,9 +371,7 @@ Follow these steps:
   On an HPC system, the correct Charm++ target architecture depends on the
   machine's inter-node communication architecture. We will be providing specific
   instructions for various HPC systems.
-* Both Charm++ and SpECTRE must be compiled using the same compiler,
-  otherwise you will receive undefined reference errors while linking SpECTRE.
-  When compiling Charm++ you can specify the compiler using, for example,
+* When compiling Charm++ you can specify the compiler using, for example,
   ```
   ./build charm++ ARCH clang
```

Comment:
what do you think about adding `-fPIC` here? That should help reduce any issues anyone encounters.